src/blog/data.json

```diff
@@ -1,4 +1,11 @@
 {
+  "introducing-ganache-7": {
+    "title": "Introducing Ganache 7",
```

Comment:
Looks like we still need the "circle" image

Reply:
Cc @CombsieZ

Reply:
I've added a temporary image so we can ship